app/views/layouts/application.html.slim

```diff
@@ -28,9 +28,10 @@ html lang="en" class="govuk-template"
 
     div class="govuk-width-container"
 
-      = yield :back
-
       main#main-content.govuk-main-wrapper role="main"
-        = yield
+        = yield :back
```

Comment:
As there's already been UAT back-and-forth over the location of the back link relative to the rest of the page, I think if we're changing the HTML structure we need to do it without changing the appearance of the link.

This shouldn't be too hard to do - I think it's a case of using a govuk utility class to reduce the margin-top of main, and then another one to add the same amount of margin-top to to the `.govuk-grid-row`